pep-0728.rst

```diff
@@ -0,0 +1,322 @@
+PEP: 728
+Title: Inner Fields Annotations
+Author: Nir Schulman <narsssx@gmail.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: https://discuss.python.org/t/26527
```

Comment:
This link should go into Post-History; a new thread should be created for PEP 728 just before this PR is merged